src/components/Home/Home.tsx

```diff
@@ -43,12 +44,14 @@ class Home extends Component {
 					</span>
 				)}>
 					<Modal.Header><Trans id="core.home.about.title">
-						About xivanalysis
+							About xivanalysis
 					</Trans></Modal.Header>
 					<Modal.Content>
 						<TransMarkdown source={about}/>
 					</Modal.Content>
 				</Modal>
+
+				<SupportSummary />
```

Comment:
Adding the summary here pushes the URL input away from the horizontal centre of the page, which might be a little confusing given how thin it is relative to the summary. This is just an idea, but maybe some css tweaks to keep the URL input centred and slightly separated from the summary? Let me know what you think.

![image](https://github.com/xivanalysis/xivanalysis/assets/65056303/32718f63-c43b-4a69-a2c9-52f644665dbe)

Reply:
Didn't have a ton of time to try to fix this before raid started today. Will see if I can cook something up tomorrow, cause I think getting some spacing in between the two elements is a good idea. Probably won't try to keep the input centered still since that causes other problems, but not having them smashed together the way they are now will be nice.